cli/worker.rs

```diff
@@ -54,11 +54,14 @@ impl CliMainWorker {
   }
 
   pub async fn setup_repl(&mut self) -> Result<(), AnyError> {
+    self.ps.prepare_node_std_graph().await?;
```

Comment:
This would be a cool feature in REPL on its own - load `deno_std` and provide it as `$std` or `std/` in REPL (like an import map)